src/ts/Implementations/Specific/AllCourses.ts

```diff
@@ -96,4 +96,12 @@ export default class AllCourses extends Page {
             this.countFinishedLessonsAndExams
         );
     }
+
+    get title(): string {
+        return this.data.title;
```

Comment:
```suggestion
        return this.manifestData.title;
```
is more immediate, available from the manifest

Reply:
Actually all pages already have a title, you can remove this whole method and it will still work